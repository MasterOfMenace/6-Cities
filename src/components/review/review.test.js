import React from 'react';
import renderer from 'react-test-renderer';
import Review from './review.jsx';

const mockReview = {
  author: `author`,
  avatar: `author-avatar`,
  text: `Review text`,
  time: new Date().toISOString()
};

it(`Правильное отображение компонента Review`, () => {
  const tree = renderer
    .create(<Review
      review={mockReview}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
