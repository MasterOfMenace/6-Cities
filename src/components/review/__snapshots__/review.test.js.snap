// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Правильное отображение компонента Review 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height="54"
        src="author-avatar"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      author
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "60%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      Review text
    </p>
    <time
      className="reviews__time"
      dateTime={2019-07-09T21:00:00.000Z}
    >
      July 2019
    </time>
  </div>
</li>
`;
